docs/concepts/policy/pod-security-policy.md

```diff
@@ -28,6 +28,7 @@ administrator to control the following:
 | Usage of host networking and ports                  | [`hostNetwork`, `hostPorts`](#host-namespaces) |
 | Usage of volume types                               | [`volumes`](#volumes-and-file-systems)      |
 | Usage of the host filesystem                        | [`allowedHostPaths`](#volumes-and-file-systems) |
+| Usage of FlexVolume drivers                         | [`allowedFlexVolumes](#flexvolume-drivers) |
```

Comment:
missing a backtick at the end of `allowedFelxVolumes` here

Reply:
ah ... will fix.